diff --git a/meson.build b/meson.build
index db8c5e5..706c9d4 100644
--- a/meson.build
+++ b/meson.build
@@ -1,6 +1,6 @@
 project('agbpack', 'c', default_options: 'c_std=gnu11')
 
-executable('agbpack', [
+agbpack = executable('agbpack', [
     'rt/out/crt0_multiboot_bin.c',
     'rt/out/crt0_rom_bin.c',
     'src/main.c',
@@ -12,4 +12,7 @@ executable('agbpack', [
     'vendor/apultra/src/libdivsufsort/lib/divsufsort_utils.c',
     'vendor/apultra/src/libdivsufsort/lib/sssort.c',
     'vendor/apultra/src/libdivsufsort/lib/trsort.c'
-], include_directories: include_directories('rt/out', 'src', 'vendor/apultra/src', 'vendor/apultra/src/libdivsufsort/include'))
+], include_directories: include_directories('rt/out', 'src', 'vendor/apultra/src', 'vendor/apultra/src/libdivsufsort/include'),
+native: true)
+
+meson.override_find_program('agbpack', agbpack)
